### PR TITLE
[Docbug][Codesampleintro][3764]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Sample code introduction corrected

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/send-proactive-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/caa01df72e52200b7aa6df4f12ce19b696af03f1/msteams-platform/bots/how-to/conversations/send-proactive-messages.md) | 76 | 75 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a3c1ba8f-4ed8-4db9-bf51-a5307231a63f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/send-proactive-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0046fa0980cd166634522c7cdc647af70ce624ef/msteams-platform/bots/how-to/conversations/send-proactive-messages.md) | 76 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/830e5143-6618-4cd5-9b08-6f35113c7445) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/conversations/send-proactive-messages.md [hunk 1]
```diff
@@ -266,6 +266,7 @@ The following table provides a simple code sample that incorporate basic convers
...
 | Start new thread in a channel | Demonstrates creating a new thread in a channel. | [View](https://github.com/microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/58.teams-start-new-thread-in-channel) | [View](https://github.com/microsoft/BotBuilder-Samples/blob/master/samples/javascript_nodejs/58.teams-start-new-thread-in-channel) | [View](https://github.com/microsoft/BotBuilder-Samples/blob/master/samples/python/58.teams-start-thread-in-channel) |
+| Proactive installation of app and sending proactive notifications | This sample shows how you can use proactive installation of app for users and send proactive notifications by calling Microsoft Graph APIs. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-proactive-installation/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-proactive-installation/nodejs) | |
```
Comment (Wajeed-msft) [inline msteams-platform/bots/how-to/conversations/send-proactive-messages.md:269]: Could you please include the same in https://review.docs.microsoft.com/en-us/microsoftteams/platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages?branch=pr-en-us-3772 doc as well?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/send-proactive-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9cddc46c654c7a94c505652026f86af2d54b418e/msteams-platform/bots/how-to/conversations/send-proactive-messages.md) | 76 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/081ab514-bfad-4ad7-a244-a94e120187e1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9cddc46c654c7a94c505652026f86af2d54b418e/msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md) | 83 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1869fe79-d852-4d98-9581-6115a092c6a1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/send-proactive-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a5e04e2f2e7e2e33b9f1bc18f704b04008d1d2d/msteams-platform/bots/how-to/conversations/send-proactive-messages.md) | 76 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a488eca8-8754-42d4-8223-b97026a87b44) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a5e04e2f2e7e2e33b9f1bc18f704b04008d1d2d/msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md) | 83 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a73bfecc-c309-4e60-9a74-67f465907cf9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**